src/schemas/Program/index.js

```diff
@@ -255,6 +255,19 @@ const resolveSingleProgram = async (egoToken, programShortName) => {
 	return response ? convertGrpcProgramToGql(programDetails) : null;
 };
 
+const resolveHTTPProgram = async (programShortName) => {
+	const response = await programService.getProgramPublicFields(programShortName);
+	return response;
+};
+
```

Comment:
Add a http public resolver in schemas. The response fetch the right information in Playground. 
<img width="1104" alt="Screenshot 2023-03-22 at 10 41 54 AM" src="https://user-images.githubusercontent.com/79996555/226940503-f6d66fc3-3c0e-4b55-aa6c-cb2421c26c60.png">
<img width="682" alt="Screenshot 2023-03-22 at 10 42 07 AM" src="https://user-images.githubusercontent.com/79996555/226940524-5b6632a7-1a2d-4655-902c-2ce61f59293e.png">